.github/workflows/tests.yaml

```diff
@@ -118,3 +118,13 @@ jobs:
       - name: Running ${{ matrix.testcase }}
         shell: bash
         run: docker compose -f tools/autograph-client/integration-tests.yml run ${{ matrix.testcase }}
+
+      - name: Fetching autograph logs
+        shell: bash
+        if: ${{ always() }}
```

Comment:
Will this cause issues if we don't reach the previous step? Maybe we don't care if this fails in weird ways if something early (eg: checking out) fails?

Reply:
If the container never starts, this just outputs nothing so the failure mode is a no-op in this case.

I also don't think it matters if the job fails in weird ways at this point as we are already dealing with an error. What's one more?